### ISSUE TITLE
Running the trained model on new images

### ISSUE THREAD

**Issue by abaner23** [MasazI/cnn_depth_tensorflow#11]
Hello,

Can anybody please tell how to run the trained model on new images?

Regards,
Arnab Banerjee

**Comment by mohanhanmo** [comment]
Do you have the trained model?

**Comment by abaner23** [comment]
Yes. I have the trained model. To train run the code given in the repository. You can use the checkpoint files and use the following code to run it on any new image:

https://github.com/barnab-msit/Depth_Tensorflow_TestImagse/blob/master/depthall1.py
